spring/boot3-webflux-autoconfigure/src/main/java/com/linecorp/armeria/spring/web/reactive/ArmeriaServerHttpResponse.java

```diff
@@ -54,7 +52,7 @@
 /**
  * A {@link ServerHttpResponse} implementation for the Armeria HTTP server.
  */
-final class ArmeriaServerHttpResponse extends AbstractServerHttpResponse {
+final class ArmeriaServerHttpResponse extends AbstractServerHttpResponseVersionSpecific {
```

Comment:
So this class now extends to two different classes depending on the module.
How about making it the opposite which means the version-specific implementation extends to  `ArmeriaServerHttpResponse`?

Reply:
It may be possible. But as you see, the name is pretty verbose. so I didn't want to use it in other places and hide it in the super class.